Apps/Examples/Examples/All Examples/Annotations/LineAnnotationExample.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+import MapboxMaps
+
+@objc(LineAnnotationExample)
+final class LineAnnotationExample: UIViewController, ExampleProtocol {
```

Comment:
![simulator_screenshot_36EF4812-C5A9-438D-9D47-C085A2CA2F5C](https://user-images.githubusercontent.com/205658/160694368-417e973a-d91c-4739-ae44-33b5815468da.png)

Can we add a note about how this clipping effect is achieved?

Reply:
Any suggestions on the content of the note? I'm not sure if I have enough understanding of this layer to put it in words.

Reply:
@Chaoba looks like you've worked on the corresponding PolylineAnnotationActivity for Android. Can you help here? How do the lines get clipped to only cover land masses?

Reply:
That's because we set the below layer id here: https://github.com/mapbox/mapbox-maps-ios/pull/1215/files#diff-133e2689b994a8088ae07857e5bf56d8284a4c10ba7df85d4af8cd0af8a70dc1R64